src/styles/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 16px 3px 0;
+}
+
+.card__title {
+  margin: 40px auto 0;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--mainTextColor);
+}
+
+.card__subtitle {
+  margin: 4px 0 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondaryTextColor);
+}
+
+.card__raiting {
+  display: flex;
+  justify-content: space-between;
+  margin: 16px 0 0;
+}
+
+.card__reviews {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--mainTextColor);
+}
+
+.card__prices {
+  margin: 24px 0 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondaryTextColor);
+}
+
+.card__value {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--mainTextColor);
+}
+
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  text-transform: uppercase;
+  margin: 16px 0 0;
+  width: 166px;
```

Comment:
we don't need define width here as well

Reply:
Sorry, but I don't understand.
In task was told: "follow styles from the mock".
Width was taken from there. 
This is not problem for me to delete width and add margin auto.
I think i need to visit Q&A meeting.

Reply:
It means that you will get the same width using right properties inside the card. Because it's not good when you hardcode height/width of an element wrapper like .card